### DISCUSSION TITLE
SMTP google

### DISCUSSION THREAD

**Discussion by Johhno** [symfony/symfony#51491] [Q&A] [answered]
Hello everyone, 

I am on Symfony 5.4 and i developed an app where people can reset their password.
LOCAL : i succeeded to receive mail with mailtrap.io but i want to deploy on production using smtp Google.

I used **MAILER_DSN=gmail://${MAIL_USER}:${MAIL_PASS}@localhost**
I am testing without **success.**

May i get help?

Regards

**Comment by gordinskiy** [comment, marked as answer]
> I am testing without success.  

What error did you get?

For Google SMTP you need to generate Application password and use it instead of regular. ([Doc](https://knowledge.workspace.google.com/kb/how-to-generate-an-app-passwords-000009237?hl=en))
